pkg/networks/commands.go

```diff
@@ -0,0 +1,84 @@
+package networks
+
+import (
+	"fmt"
+	"github.com/lima-vm/lima/pkg/store"
+)
+
+const (
+	Switch = "switch"
```

Comment:
For performance reason, we may just ditch the switch and use PTP mode everywhere: https://github.com/lima-vm/vde_vmnet/issues/26#issuecomment-917392360

That may also beneficial for simplicity.

Reply:
I would add support for PTP mode in a later PR; I believe it can be added as just another network, albeit one that can only be used by one instance at a time.

And e.g. a host-only network doesn't work in PTP mode because you can't connect multiple instances to it. Maybe it doesn't make sense to consider this, but I was thinking about being able have one instance on a host-only network, and a second instance running as a proxy server/jumpbox, to test that an application works correctly in a fully proxied environment.